candidate-standard/abstract_tests/instances/ATS_src-md-success.adoc

```diff
@@ -0,0 +1,8 @@
+[[ats_instances_src-md-success]]
+[width="90%",cols="2,6a"]
+|===
+^|*Abstract Test {counter:ats-id}* |*/ats/instances/src-md-success*
+^|Test Purpose |Validate that the Instance of a Collection content complies with the required structure and contents.
+^|Requirement |<<req_collections_src-md-success,/req/collections/src-md-success>>
+^|Test Method |Verify that the content of the response is consistent with the content for this Resource Collection in the `/collections` response. That is, the values for `id`, `title`, `description` and `extent` are identical.
```

Comment:
For instances the `extent` must be within the bounds of the `extent` of the collection.  In the case of model data each **instance** would represent a different model run, so the **collection** time coordinates would cover the time span for all instances but each individual **instance** would be a subset of that time range.

Reply:
@m-burgoyne We have a different interpretation of an instance. I am happy to take a looser definition, so that Instances and instanceId parallel exactly Collections and collectionId.
Lets discuss and close in Issue #77